docs/performance/tune-performance.md

```diff
@@ -169,7 +169,39 @@ Currently, there are no special provisions to employ mimalloc on Linux. It is re
 
 ### Thread management
 
-* Use the appropriate ORT API to set intra and inter op num threads. Inter op num threads is only used when parallel execution is enabled.
+#### Set number of intra-op threads
```

Comment:
We should add a separate section to call out support for NUMA.

Reply:
Added separate section for NUMA.